composer/utils/collect_env.py

```diff
@@ -348,7 +348,7 @@ def print_env(file: Optional[TextIO] = None) -> None:
 
         Composer information
         --------------------
-        Composer version: 0.8.0
+        Composer version: 0.4.2
```

Comment:
Instead of trying to keep this up to date, let's make it a meme...

Reply:
Yes you could do:
```
__doc__ = """blah blah {blah}""".format(blah='blah')
```

But this is fine too :)

Reply:
@hanlint just out of curiosity, is python smart enough to unroll the constants like that at function definition time rather than at function evaluation?